### DISCUSSION TITLE
[Weekly Meeting] [2024-07-31] - Meeting Notes

### DISCUSSION THREAD

**Discussion by ricardozanini** [serverlessworkflow/specification#942] [Meetings Notes]
# Topics

- [x] @ricardozanini to [configure gitvote](https://github.com/cncf/gitvote/blob/main/docs/config/.gitvote.yml). We need to go through the options (not necessarily in the meeting) and see what fits best. The application has been installed in the `specification` project. Kudos to @cdavernas for bringing the idea! 💡 
- [x] Re-adding the `metadata` property to cover edge cases and UI implementations to add information regarding element data
- [x] Elaborate an idea for the Spec recommendations regarding UI. Example: "use `metadata` to hold your value"; "add an option to export the flow diagram to SVG"; etc.

**Comment by cdavernas** [comment]
- [x] Name and create the repository for custom functions
- [x] Decide which custom functions to officially support (ex: log, read/write file, ...)

**Comment by ricardozanini** [comment]
## Actions Items

- [x] @ricardozanini To open an issue to introduce `meta` to certain concepts and link to https://github.com/serverlessworkflow/specification/pull/928
- [x] @ricardozanini To have a special discussion category for "official" FAQ with questions from the community (either promoted from discussions or open based on Slack threads)
- [ ] @cdavernas To have guidelines and best-practices documentation in the spec. For example: use `meta` to hold UI elements position on a GUI implementation; avoid using `meta` to change behavior of a runtime execution.
- [x] @JBBianchi to add a README to the organization's front page. Send a PR here: https://github.com/serverlessworkflow/.github
- [x] @ricardozanini to open a new X account for the project and share the credentials on 1Password.
- [x] The new catalog repository has been added: https://github.com/serverlessworkflow/catalog
- [x] Modified the [organization](https://github.com/serverlessworkflow) landing page's pins: removed Synapse and added Catalog
- [x] Updated the description of the `specification` repository
- [x] postpone: Decide which custom functions to officially support (ex: log, read/write file, ...)

## Decisions

- New repo for custom functions will be named `catalog`. An example of a catalog: https://github.com/tektoncd/catalog/tree/main
- To start marketing our things cross-media (X, LinkedIn, Blog). @JBBianchi brought an idea to have a CI/CD action on the website to post on X and LinkedIn once a Blog post is merged.

**Reply by ricardozanini** [reply]
Regarding the "official" FAQ I'll postpone until we have enough discussions around the guidelines/best practices too instead of creating an empty category for now.
